### PR TITLE
Add test cases for workspace creation

### PR DESCRIPTION
**Pull request by Kapian1234**
### Description
1. Create a workspace from home page
2. Jump to collaborators page after creating a workspace.
3. The card on the right summarizes the information.

### Issues Resolved

[List any issues this PR will resolve]

### Check List

- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_create.spec.js [hunk 1]
```diff
@@ -91,6 +91,96 @@ if (Cypress.env('WORKSPACE_ENABLED')) {
...
+          'workspace-initial-button-create-observability-workspace'
+        ).click({
+          force: true,
+        });
+        cy.contains('Observability')
+          .first()
+          .closest('.euiCheckableCard-isChecked')
+          .should('exist');
+
+        miscUtils.visitPage('app/workspace_initial');
```
Comment (wanglam) [inline cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_create.spec.js:113]: Why we need to visit initial page twice?

File: cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_create.spec.js [hunk 2]
```diff
@@ -91,6 +91,96 @@ if (Cypress.env('WORKSPACE_ENABLED')) {
...
+
+          const expectedWorkspace = {
+            name: workspaceName,
+            features: ['use-case-security-analytics'],
+          };
+          cy.checkWorkspace(workspaceId, expectedWorkspace);
+        });
+      });
+
+      it('should successfully jump to collaborators page after creating a workspace', () => {
```
Comment (wanglam) [inline cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_create.spec.js:148]: Prefer adding SECURITY_ENABLED and SAVED_OBJECTS_PERMISSION_ENABLED check before this test case.

File: cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_create.spec.js [hunk 3]
```diff
@@ -91,6 +91,96 @@ if (Cypress.env('WORKSPACE_ENABLED')) {
...
+        ).type('test_workspace_description.+~!');
+        cy.getElementByTestId('workspaceUseCase-essentials').click({
+          force: true,
+        });
+        inputDataSourceWhenMDSEnabled(dataSourceTitle);
+        cy.get('.workspaceCreateRightSidebar').within(() => {
+          cy.contains(workspaceName).should('exist');
+          cy.contains('test_workspace_description.+~!').should('exist');
+          cy.contains('Essentials').should('exist');
+          cy.contains(dataSourceTitle).should('exist');
```
Comment (wanglam) [inline cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_create.spec.js:181]: Need to add MDSEnabled check here.

File: cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_create.spec.js (hunk 1 above)
Comment (Kapian1234) [reply cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_create.spec.js:113]: to check if both these buttons could navigate to the create page
Screenshot 2024-11-13 at 11 39 40

File: cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_create.spec.js (hunk 2 above)
Comment (Kapian1234) [reply cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_create.spec.js:148]: ok

File: cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_create.spec.js (hunk 3 above)
Comment (Kapian1234) [reply cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_create.spec.js:181]: thanks

File: cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_create.spec.js (hunk 1 above)
Comment (wanglam) [reply cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_create.spec.js:113]: I don't think the workspace create integration test should cover this part. But it's OK. We can keep this.

**Review by SuZhou-Joe — APPROVED** [review]

**Review by wanglam — APPROVED** [review]

**Pull request merged**